support-frontend/.storybook/main.js

```diff
@@ -10,38 +10,85 @@ module.exports = {
 		'@storybook/addon-storysource',
 		'@storybook/addon-viewport',
 		'@storybook/addon-interactions',
-		'@storybook/addon-webpack5-compiler-babel',
 	],
+	core: {
+		builder: '@storybook/builder-vite',
+	},
 	framework: {
-		name: '@storybook/preact-webpack5',
+		name: '@storybook/preact-vite',
 		options: {},
 	},
 	features: {
 		interactionsDebugger: true,
 	},
 	staticDirs: ['./static'],
-	webpackFinal: async (config) => {
-		return {
-			...config,
+	viteFinal: async (config, { configType }) => {
+		return vite.mergeConfig(config, {
 			plugins: [
-				...config.plugins,
-				new MiniCssExtractPlugin({
-					filename: path.join('stylesheets', 'style.css'),
+				preact.preact({
+					reactAliasesEnabled: true,
+					jsxImportSource: '@emotion/react',
 				}),
 			],
-			module: {
-				...config.module,
-				rules: devConfig.module.rules,
-			},
 			resolve: {
-				...config.resolve,
-				alias: {
-					...devConfig.resolve.alias,
-					'@modules': path.resolve(__dirname, '../../modules'),
-				},
-				modules: devConfig.resolve.modules,
+				alias: [
```

Comment:
Presume just easier to read over creating a new `webpack.dev.js` equivalent `vite.dev.js` file?